src/main/resources/templates/webshop/registration/study.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head></head>
+<body>
+<!--/*@thymesVar id="study" type="ch.wisv.events.core.model.registration.Study"*/-->
+<div th:fragment="study">
+    <div class="card mb-4">
+        <h6 class="card-header">Study</h6>
+        <div class="card-body">
+            <div class="form">
+                <div class="form-group">
+                    <label>Study*
+                        <select class="form-control" th:field="*{study.studyName}" required>
+                            <option value="Bachelor Computer Science and Engineering">Bachelor Computer Science
+                                and Engineering
+                            </option>
+                            <option value="Bachelor Applied Mathematics">Bachelor Applied Mathematics</option>
+                            <option value="Double Bachelor (AP /AM)">Double Bachelor (AP /AM)</option>
+                            <option value="Master Applied Mathematics">Master Applied Mathematics</option>
+                            <option value="Master Computer Science">Master Computer Science</option>
+                            <option value="Master Embedded Systems">Master Embedded Systems</option>
+                            <option value="Master Computer Engineering">Master Computer Engineering</option>
+                        </select>
+                    </label>
+                </div>
+                <div class="form-group">
+                    <label>First study year*
+                        <input type="number" min="2014" class="form-control"
```

Comment:
Is this a dropdown? Maybe minimum year = this year minus 4 in that case instead of arbitrary 2014?